databricks_cli/sdk/service.py

```diff
@@ -997,29 +997,43 @@ def deploy(self, pipeline_id=None, id=None, name=None, storage=None, configurati
                 raise TypeError('Expected databricks.Filters() or dict for field filters')
         if allow_duplicate_names is not None:
             _data['allow_duplicate_names'] = allow_duplicate_names
-        return self.client.perform_query('PUT', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id), data=_data, headers=headers)
-    
+        return self.client.perform_query('PUT', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id), data=_data,
+                                         headers=headers)
+
     def delete(self, pipeline_id=None, headers=None):
         _data = {}
-    
-        return self.client.perform_query('DELETE', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id), data=_data, headers=headers)
-    
+
+        return self.client.perform_query('DELETE', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id),
+                                         data=_data, headers=headers)
```

Comment:
Is this spacing what got generated or a copy/paste error?

Reply:
Yup, it's all generated.

Reply:
I'll revert of it except for the `list` function - it unnecessarily bloats this PR.